src/main/java/net/p455w0rd/wirelesscraftingterminal/client/gui/GuiCraftAmount.java

```diff
@@ -112,8 +110,14 @@ public void drawBG(final int offsetX, final int offsetY, final int mouseX, final
         this.drawTexturedModalRect(offsetX, offsetY, 0, 0, this.xSize, this.ySize);
 
         try {
-            Long.parseLong(this.amountToCraft.getText());
-            this.next.enabled = this.amountToCraft.getText().length() > 0;
+            String out = this.amountToCraft.getText();
+
+            double resultD = Calculator.conversion(out);
+            int resultI;
```

Comment:
Now that you extracted it, why separate the declaration and assignment

Reply:
eh true